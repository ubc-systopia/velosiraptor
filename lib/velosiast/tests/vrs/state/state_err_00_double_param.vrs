state(base: addr, base: addr) {
    reg foo [ 8 ]
}