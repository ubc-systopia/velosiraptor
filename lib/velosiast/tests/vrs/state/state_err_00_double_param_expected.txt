error: duplicate definitions with name `base`
     --> tests/vrs/state/state_err_00_double_param.vrs:1:30
      |
    1 |  state = StateDef(base: addr, base: addr) {
      |                   ^^^^^^^^^^ previous definition of `base` was here
   ...
      |
    1 |  state = StateDef(base: addr, base: addr) {
      |                               ^^^^^^^^^^ duplicate definition
error: could not compile due to 1 previous errors; 0 warnings emitted


@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
